automation/lib/auth.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+function authenticate () {
+   echo $1 | gh auth login --with-token
+   gh auth setup-git
+}
+
+function config-git () {
+    git config user.name ""$(gh api user -q ".login")""
+    git config user.email "trestle@example.com"
```

Comment:
Should we allow this to be configured? GitHub will not be able to find a user associated with this email.